android/build.gradle

```diff
@@ -56,8 +60,6 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
     implementation "androidx.appcompat:appcompat:$androidxAppCompatVersion"
-    implementation 'com.hypertrack:hypertrack:6.2.0'
-    testImplementation "junit:junit:$junitVersion"
-    androidTestImplementation "androidx.test.ext:junit:$androidxJunitVersion"
-    androidTestImplementation "androidx.test.espresso:espresso-core:$androidxEspressoCoreVersion"
+    implementation 'com.hypertrack:hypertrack:6.4.0'
+    implementation 'com.google.firebase:firebase-messaging:23.1.1'
```

Comment:
I would put them in variables like `androidxAppCompatVersion`
BTW, there are two different sets of variables:
line 9: `ext.kotlin_version = "1.6.21"` 
and the ones on the top of this file

Reply:
Ext executes after buildscript and is part of the autogenerated Ionic stuff